BUILD

```diff
@@ -0,0 +1,818 @@
+load("@llvm-project//mlir:tblgen.bzl", "gentbl_cc_library", "gentbl_filegroup", "td_library")
```

Comment:
can we name this BUILD.bazel instead of BUILD? It makes it totally obvious what the file is for, syntax highlighting etc.

Same for WORKSPACE.bazel.

Reply:
Done!

Reply:
Thanks. Really important to avoid naked BUILD and WORKSPACE files (i wish the bazel folks would deprecate them): on case insensitive file systems, which includes Macos, both bare names tend to conflict with directories that sometimes need to exist (ie. Blocks a sibling build/ directory, which can't even be changed for pip builds).